src/autogluon/bench/cloud/aws/destroy.sh

```diff
@@ -16,9 +16,9 @@ IMAGE_URI=$(echo "${OUTPUTS_JSON}" | jq -r '.[] | select(.OutputKey == "ImageUri
 IMAGE_TAG_OR_DIGEST=$(echo "${IMAGE_URI}" | sed -n 's/.*://p')
 
 echo "Destroying $BATCH_STACK_NAME"
-cdk destroy --app $CDK_PATH $BATCH_STACK_NAME
+cdk destroy --app $CDK_PATH $BATCH_STACK_NAME --force
 echo "Destroying $STATIC_RESOURCE_STACK_NAME"
-cdk deploy --app $CDK_PATH $STATIC_RESOURCE_STACK_NAME
+cdk deploy --app $CDK_PATH $STATIC_RESOURCE_STACK_NAME --force
```

Comment:
What kind of issues are you expecting that `--force` is always required?

Reply:
`--force` works similarly as `--require-approval never` as above. They have a slightly different interface for `cdk deploy` and `cdk destroy`